app/models/fiken/address.rb

```diff
@@ -0,0 +1,5 @@
+module Fiken
+	class Address
```

Comment:
Hva trenger vi denne til ?

Reply:
Den er en del av Contact-objektet. Kan sikkert fjernes, men jeg ville heller ha en tom type for det enn et ikke-komplett sett med felter på contact.